controllers/extendeddaemonset_e2e_test.go

```diff
@@ -465,7 +466,7 @@ var _ = Describe("ExtendedDaemonSet e2e successful canary deployment update", fu
 				},
 			}
 
-			eds := testutils.NewExtendedDaemonset(namespace, name, "k8s.gcr.io/pause:latest", edsOptions)
+			eds := testutils.NewExtendedDaemonset(namespace, name, "k8s.gcr.io/pause:3.0", edsOptions)
```

Comment:
I think you can use the new kubernetes registry image

```
➜ crane ls  registry.k8s.io/pause
0.8.0
1.0
2.0
3.0
3.1
3.2
3.3
3.4.1
3.5
3.6
3.7
3.8
3.9
```

Reply:
but not `latest` because the tag is not updated

Reply:
Alright. Will update the used image. Thank you